Procfile

```diff
@@ -1,4 +1,4 @@
-web: gunicorn kuma.wsgi -b 0.0.0.0:8000
+web: gunicorn kuma.wsgi -w 2 -b 0.0.0.0:8000
```

Comment:
Yes, it appears the user's request "GET page" (actually POST new page) was blocking KumaScript's "Fetch Raw Source" request:

![KumaScript overview](https://wiki.mozilla.org/images/thumb/2/2b/Kumascript.png/1000px-Kumascript.png)